source/assets/stylesheets/ie8_fixes.scss

```diff
@@ -31,12 +33,23 @@
 }
 
 // Place icons in the middle of its container
-.footer-social_icons__icon {
+.social_icons__icon {
   padding-top: 6px;
   height: 22px;
 }
 
 // To support full coloured div
-.footer-social-icons {
+.social-icons {
   display: block;
 }
+
+// To set a blog article width
+.blog-header__article-header,
+.blog-article__container,
+.blog-author__container,
+.blog-comments__container,
+.blog-related__container,
+.blog-related__header {
```

Comment:
Trying to understand why these are only set for IE8..?

Also, we could have one main container for the blog post (maybe the `blog-page` class) that sets these properties? I understand there is a slightly different container in the related section so we could have an alternative container style.